sphinx/tutorial/cylc/scheduling/datetime-cycling.rst

```diff
@@ -46,6 +46,17 @@ For readability we may add hyphen (``-``) characters between the date
 components and colon (``:``) characters between the time components, i.e:
 ``2000-01-01T00:00``.
 
+.. warning::
+
+   Hyphen and colon characters can be added to a date-time,
+   or can be omitted, depending on preference, but *both* the date and time
+   components must have the appropriate characters inserted, should
+   characters be used.
```

Comment:
We can probably simplify this, it might be a good idea to convey that these are the "basic" and "extended" formats.

```diff
 For readability we may add hyphen (``-``) characters between the date
 components and colon (``:``) characters between the time components, i.e:
-``2000-01-01T00:00``.
+``2000-01-01T00:00``. This is the "extended" format.

+ .. warning::
+
+    The "basic" (plain) and "extended" (written with hyphens and colons)
+     formats cannot be mixed. For example the following date-times are invalid:
+
+    .. code-block:: none
+
+       2000-01-01T0000
+       20000101T00:00
+
-.. warning::
-
-   Hyphen and colon characters can be added to a date-time,
-   or can be omitted, depending on preference, but *both* the date and time
-   components must have the appropriate characters inserted, should
-   characters be used.
-
-   Date-times such as ``2000-01-01T0000`` or ``20000101T00:00``, with
-   characters inside only one such component, are invalid in :term:`ISO8601`
-   and hence will produce an error.
```

Reply:
Yes, that is much better (assuming the indentation is fixed). Your documentation-writing experience is showing here as that is clearer and twice as concise! I'll change the file accordingly.